doc/source/io.rst

```diff
@@ -2867,6 +2867,23 @@ indices to be parsed.
 
 Element order is ignored, so ``usecols=[0, 1]`` is the same as ``[1, 0]``.
 
+If ``usecols`` is a list of strings, it is assumed that each string corresponds
```

Comment:
need a versionadded / changed here? or is this just added documentation?

Reply:
Ah, good point.  It should have version added tags.